modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/ModelController.java

```diff
@@ -43,25 +44,26 @@
         return modelService.getModels(metricKey);
     }
 
-    @ApiOperation(value = "Add model hyper params")
-    @PostMapping(value = "/addModelParams", produces = MediaType.APPLICATION_JSON_VALUE)
-    public String addModelParams(@RequestBody ModelParams modelParams) {
-        modelService.addModelParams(modelParams);
+    @ApiOperation(value = "Add hyperparams")
+    @PutMapping(value = "/models/{uuid}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public String addHyperparams(@PathVariable String uuid, @RequestBody Hyperparams hyperparams) {
+        modelService.addModelParams(uuid, hyperparams);
         return "Model params saved successfully";
     }
 
     @ApiOperation(value = "Mark model to rebuild")
-    @PutMapping(value = "/markToRebuild", produces = MediaType.APPLICATION_JSON_VALUE)
-    public String markToRebuild(@RequestBody RebuildParams rebuildParams) {
-        modelService.markToRebuild(rebuildParams.getModelUUID(), rebuildParams.getMetricKey(),
-                rebuildParams.getToRebuild());
+    @PutMapping(value = "/rebuildModel/{uuid}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
REST URIs should be nouns, not verbs. (Use `/models/:uuid` rather than `/rebuildModel/:uuid`. See previous PR for more info.

Also we need to store the rebuild schedule, right? E.g. rebuild every 24 hours or whatever. Would it be better to store the last scheduled rebuild time and the rebuild period, and then do schedule a rebuild whenever now <= lastScheduled + period? That way we don't require some external actor to come in and set this `toRebuild` flag.